packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -545,6 +544,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         requestHandler?: (request: IRequest, runtime: IContainerRuntime) => Promise<IResponse>,
         runtimeOptions?: IContainerRuntimeOptions,
         containerScope: IFluidObject = context.scope,
+        existing?: boolean,
```

Comment:
Assuming this chance alone goes in before any other changes (i.e. it will be in next release branch), I'm not sure developers consuming FF (like Bohemia) have enough guidance given this change on what to pass in here.
Argument being optional helps (it will not trigger build breaks), but at least we need to @param description above RE expectations.